__tests__/db.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest');
+const app = require('../db/app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+afterAll(() => db.end());
+
+beforeEach(() => seed(testData));
+
+describe('app', () => {
+  describe('topics', () => {
+    test('api/topics - responds with all topics and 200 status code', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((res) => {
+          expect(res.body.topic.length).toBe(3);
```

Comment:
Great test here! checking the array length matches what we expect before your forEach stops us getting any false positives if the array was empty 😄